lib/FieldTags/FieldTagsContainer.js

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+
+import { useShowToast } from '../hooks';
+import { baseManifest } from '../manifests';
+import FieldTags from './FieldTags';
+
+const FieldTagsContainer = ({
+  formName,
+  mutator,
+  name,
+  resources,
+  stripes: { store },
+}) => {
+  const allTags = get(resources, ['tags', 'records'], []);
+  const tagSettings = get(resources, ['configTags', 'records'], []);
+  const tagsEnabled = !tagSettings.length || tagSettings[0].value === 'true';
+
+  const showCallout = useShowToast();
```

Comment:
Is it OK to use it here? Wouldn't it be a tight coupling to the code of our Callout implementation? 
cc @NikitaSedyx

Reply:
it works

Reply:
works, but without context it won't
it can be hidden bug when this component is used without provider and callout set up

Reply:
removed `Callout`